packages/catalog-model/src/kinds/TemplateEntityV1alpha1.test.ts

```diff
@@ -90,4 +91,8 @@ describe('templateEntityV1alpha1Validator', () => {
     (entity as any).spec.templater = '';
     await expect(validator.check(entity)).rejects.toThrow(/templater/);
   });
+  it('accepts missing owner', async () => {
+    delete (entity as any).spec.owner;
+    await expect(validator.check(entity)).resolves.toBe(true);
+  });
```

Comment:
tests to add: rejects empty owner (`''`), rejects wrong type (e.g. `7`)

Reply:
@freben So, must `owner` prop be required?

Reply:
No, it can't be (it will break existing entities out in the wild). So the tests will just check that IF the owner is set, it must not be the empty string, and it must not be of another type than a string